### PR TITLE
Removed all the deprecated fields and their associated codes.

### PR DESCRIPTION
**Pull request by harshitasao**
Part of  #19264

### CODE REVIEW

**Comment by matthyx** [comment]
Welcome @harshitasao !
/ok-to-test

**Comment by matthyx** [comment]
@harshitasao I don't think you can modify `prow/ANNOUNCEMENTS.md` anymore.

**Comment by harshitasao** [comment]
My apologies, @matthyx could you please guide me through what I should do next?

**Comment by matthyx** [comment]
> My apologies, @matthyx could you please guide me through what I should do next?

Just revert your changes to this particular file... I am not sure if we need an announcement for that task anyways.

**Comment by matthyx** [comment]
cheers!
/lgtm
/assign @chaodaiG

File: prow/plugins/config.go [hunk 1]
```diff
@@ -1300,8 +1292,8 @@ func (c *Configuration) Validate() error {
 		logrus.Warn("no plugins specified-- check syntax?")
 	}
 
-	if c.Owners.LabelsBlackList != nil && c.Owners.LabelsDenyList != nil {
-		return errors.New("labels_blacklist and labels_denylist cannot be both supplied")
+	if c.Owners.LabelsDenyList != nil {
+		return errors.New("labels_denylist cannot be supplied")
```
Comment (chaodaiG) [inline prow/plugins/config.go:1296]: this might be incorrect. `labels_denylist` could be supplied, am I missing something?

**Review by chaodaiG — CHANGES_REQUESTED** [review]
Great work! one comment

File: prow/plugins/config.go (hunk 1 above)
Comment (harshitasao) [reply prow/plugins/config.go:1296]: Thank you for bringing this up. I think `labels_denylist` could be supplied. So I will make the required changes.

**Pull request merged**